packages/dashboard/src/components/CreateWalletForm/index.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+    FormControl,
+    ListItemIcon,
+    MenuItem,
+    Select,
+    experimentalStyled as styled,
+    Typography,
+    makeStyles,
+    createStyles,
+    FilledInput,
+} from '@material-ui/core'
+import { memo, useState } from 'react'
+
+const useStyles = makeStyles(() =>
+    createStyles({
+        root: {
+            width: 380,
+            marginTop: 12,
+        },
+        input: {
+            paddingTop: 15,
+            paddingBottom: 15,
+        },
+    }),
+)
+
+// TODO: actions,  and icon may be an img url
+export interface CreateWalletFormProps {
+    options: Array<{ label: string; icon: React.ReactNode; value: number }>
+}
+
+export const CreateWalletForm = memo((props: CreateWalletFormProps) => {
+    const { options } = props
+
+    const classes = useStyles()
+
+    const [selected, setSelected] = useState()
+
+    return (
+        <Container>
+            <FormContainer>
+                <Select variant="filled" value={selected} onChange={(event) => setSelected(event.target.value)}>
+                    {options.map(({ label, icon, value }) => (
+                        <MenuItem value={value}>
+                            <ListItemIcon>{icon}</ListItemIcon>
+                            <Typography variant="inherit">{label}</Typography>
+                        </MenuItem>
+                    ))}
+                </Select>
+            </FormContainer>
+
+            <FilledInput classes={classes} placeholder="Wallet Name" />
+        </Container>
+    )
+})
+
+const Container = styled('div')`
+    display: flex;
+    flex-direction: column;
+`
+
+const FormContainer = styled(FormControl)`
+    width: 380px;
+    // Because can't use makeStyles override Input-base style, So do this.
```

Comment:
You can use makeStyles for now.

Reply:
Ok, actually there is nothing to do with makeStyled and styled here, the problem I encountered is that I need to set the property on MuiFilledInput-root, but MUI does not provide the rule name to set.

Reply:
Isn't there anything like `<FormControl InputProps={{ classes: { root: {} } }} />`?

Reply:
No, not provided here